### DISCUSSION TITLE
A forge 1.16.5 port

### DISCUSSION THREAD

**Discussion by Derailedzack** [Ladysnake/Requiem#321] [Q&A] [answered]
Should this mod be ported to forge 1.16.5?

**Comment by Derailedzack** [comment]
If no one wants this to be ported to 1.16.5 forge then I'll work on something else

**Reply by grayWolfGameingNo2** [reply]
Hello Derailedzack, Are you still working on the 1.16.5 port? Me along with a lot of people would love to have this port for their server/moddpack/solo-play!

**Comment by Derailedzack** [comment, marked as answer]
Also I have already started porting it.

**Reply by Pyrofab** [reply]
Alright have fun ! If you manage to complete it, tell us so we can maybe link to it on our page.

**Comment by Derailedzack** [comment]
https://github.com/Derailedzack/Derailedzack/discussions/2 
This is the list of projects that are planned or in development. This doesn't have everything.

**Reply by AFANACIY** [reply]
Maybe im late(3 years have passed) but did u just gave up? WE need this mod on forge, why isn't it still ported

**Reply by Derailedzack** [reply]
That port was for 1.16.5 forge not any of the newer releases of forge. I likely still have the source for it somewhere

**Reply by Derailedzack** [reply]
If need be I can try porting Requiem to forge. What version(s) of Minecraft should the port target?

**Reply by AFANACIY** [reply]
If u can, please port it to 1.20.1 and 1.16.5, primarily 1.20.1 u are a hero bro

**Reply by AFANACIY** [reply]
Hey, just wanna ask, how long will porting take? Just to know, a week or later

**Reply by AFANACIY** [reply]
> If need be I can try porting Requiem to forge. What version(s) of Minecraft should the port target?

yo its me again, so what about requiem port? did u find 1.16.5 version or still na?

**Reply by AFANACIY** [reply]
> If need be I can try porting Requiem to forge. What version(s) of Minecraft should the port target?

Long time no see, maybe u are buzy, but ima still waiting is there port for requiem 1.16.5? I think it will be my last comment of you don't answer, cuz i dont wanan annoy you😭

**Reply by Derailedzack** [reply]
For the 1.16.5 port I would need to either port all of the libraries used in it or reimplement their functionality before even porting requiem to 1.16.5

**Reply by AFANACIY** [reply]
Oh, it's kinda hard for me to understand, but how much time will it take? Can u actually do this?
